akka-cluster-sharding/src/main/scala/akka/cluster/sharding/ShardRegion.scala

```diff
@@ -678,8 +675,8 @@ private[akka] class ShardRegion(
     coordinatorSelection.foreach(_ ! registrationMessage)
     if (shardBuffers.nonEmpty && retryCount >= 5)
       log.warning(
-        "Trying to register to coordinator at [{}], but no acknowledgement. Total [{}] buffered messages.",
-        coordinatorSelection, shardBuffers.totalSize)
+        "Trying to register to coordinator at [{}], but no acknowledgement. Total [{}] buffered messages. Members [{}]",
+        coordinatorSelection, shardBuffers.totalSize, membersByAge)
```

Comment:
There is another open PR https://github.com/akka/akka/pull/24906 that is improving this log message, so perhaps revert this change. I'm not sure we should log all members here.

Reply:
k, will remove